fog/test-client/Cargo.toml

```
@@ -40,6 +40,7 @@ lazy_static = "1.4"
 maplit = "1.0"
 more-asserts = "0.3"
 once_cell = "1.15"
+rand = "0.7"
```

Nit: `rand` is currently at 0.8.5

```suggestion
rand = "0.8"
```